test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -474,7 +474,7 @@ public async Task TrySyncTriggers_NoDurableTaskHub_UsesBundles_V1DefaultsPosted(
                 Assert.True(syncResult.Success, "SyncTriggers should return success true");
                 Assert.True(string.IsNullOrEmpty(syncResult.Error), "Error should be null or empty");
 
-                VerifyResultWithCacheOn(expectedTaskHub: null, connection: null);
+                VerifyResultWithCacheOn(expectedTaskHub: "TestHubValue", connection: null);
```

Comment:
This test assumed that bundles meant we're using DF V1, so I changed it accordingly.